UPDATE.md

```diff
@@ -14,44 +14,54 @@ The new default configuration looks something like this:
     "makeFile": "drupal.make.yml"
   },
   "sync": {
-    "directories": {
-      "root": {
-        "destination": "",
-        "method": "symlink",
-        "conflict": "overwrite"
-      },
-      "modules/custom": {
-        "destination": "modules/custom",
-        "method": "symlink",
-        "conflict": "overwrite"
-      },
-      "themes/custom": {
-        "destination": "themes/custom",
-        "method": "symlink",
-        "conflict": "overwrite"
-      },
-      "files": {
-        "destination": "sites/default/files",
-        "method": "symlink",
-        "conflict": "overwrite"
-      }
+    "root": {
+      "destination": "",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "dir",
+      "hook": "postBuild"
+    },
+    "modules/custom": {
+      "destination": "modules/custom",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "dir",
+      "hook": "preBuild"
+    },
+    "themes/custom": {
+      "destination": "themes/custom",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "dir",
+      "hook": "postBuild"
     },
     "files": {
-      "settings/settings.php": {
-        "destination": "sites/default/settings.php",
-        "method": "symlink",
-        "conflict": "overwrite"
-      },
-      "settings/secret.settings.php": {
-        "destination": "sites/default/secret.settings.php",
-        "method": "symlink",
-        "conflict": "overwrite"
-      },
-      "settings/local.settings.php": {
-        "destination": "sites/default/local.settings.php",
-        "method": "symlink",
-        "conflict": "overwrite"
-      }
+      "destination": "sites/default/files",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "dir",
+      "hook": "postBuild"
+    },
+    "settings/settings.php": {
+      "destination": "sites/default/settings.php",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "file",
+      "hook": "postBuild"
+    },
+    "settings/secret.settings.php": {
+      "destination": "sites/default/secret.settings.php",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "file",
+      "hook": "postBuild"
+    },
+    "settings/local.settings.php": {
+      "destination": "sites/default/local.settings.php",
+      "method": "symlink",
+      "conflict": "overwrite",
+      "type": "file",
+      "hook": "postBuild"
```

Comment:
So you have local.settings.php but you don't have `sites/develop.services.yml` or `sites/services.yml` for d8. These are either downloaded with the [Drupal composer project](https://github.com/drupal-composer/drupal-project) or as a part of drupal core when using Drush make, but it is something that is customized (I don't know exactly how often) What you think?